dts/bindings/ethernet/ethernet,fixed-link.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2020, DENX Software Engineering GmbH
+#		      Lukasz Majewski <lukma@denx.de>
+# SPDX-License-Identifier: Apache-2.0
+
+description: Ethernet fixed link configuration for PHY communication
+
+compatible: "ethernet,fixed-link"
+
+properties:
+    speed:
+      type: int
+      required: true
+      description: The speed of fixed link
+      enum:
+           - 100
+           - 10
+
+    full-duplex:
+      type: boolean
+      required: false
+      description: The fixed link operates in full duplex mode
+
+    half-duplex:
```

Comment:
I don't think that "half-duplex" binding is required. The driver should check for "full-duplex" and fallback to half duplex if the property is unavailable.. This seems superfluous.. 

I just looked through the standard device tree bindings for fixed-link and they say the same thing as well (https://android.googlesource.com/kernel/common.git/+/android-trusty-4.4/Documentation/devicetree/bindings/net/fixed-link.txt)

Reply:
Ok. Thanks for pointing this out - I will follow this recommendation.